doc/README.md

```diff
@@ -0,0 +1,11 @@
+Quick start guide:
+
+- Install Hugo; follow the guidance on https://gohugo.io/getting-started/installing.
+  You'll need Go as well: see https://go.dev/
+- Run a local server: `hugo server`
+- Open a browser at http://127.0.0.1:1313/
+- Add some content.
+  - The menu hierarchy comes mostly from the directory structure in `content/`.
```

Comment:
I would suggest to stick with wrapped paragraphs in Markdown documents to make review and diffs easier.

Reply:
Yes I think I've done that. Do you mean adding this guidance to the readme?